public/app/features/dashboard/api/dashboard_api.ts

```diff
@@ -42,8 +43,9 @@ class LegacyDashboardAPI implements DashboardAPI {
   }
 
   getDashboardDTO(uid: string): Promise<DashboardDTO> {
-    const scopes = config.featureToggles.passScopeToDashboardApi ? getSelectedScopesNames() : [];
-    const queryParams = scopes.length > 0 ? { scopes } : undefined;
+    const scopes = getScopes();
+    const timeRangeAndFilters = getTimeRangeAndFilters();
```

Comment:
this should not be a global function, but should take time range and filters from the scene

Reply:
you can have two dashboards open at the same time, so not relying on global state is very important